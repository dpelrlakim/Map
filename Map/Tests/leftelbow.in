[] a 0
[] a -10
[] a -20
[] a -19
[] a -18
[] a -17
[] a -16
[] a -9
[] a -8
[] a -7
erase a -20
erase a -19
erase a -18
erase a -17
erase a -10
erase a -16
erase a -9
erase a -8
erase a 0
erase a -7
