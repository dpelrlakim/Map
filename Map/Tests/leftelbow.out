0
a: (0,0)
0
a: (-10,0) (0,0)
0
a: (-20,0) (-10,0) (0,0)
0
a: (-20,0) (-19,0) (-10,0) (0,0)
0
a: (-20,0) (-19,0) (-18,0) (-10,0) (0,0)
0
a: (-20,0) (-19,0) (-18,0) (-17,0) (-10,0) (0,0)
0
a: (-20,0) (-19,0) (-18,0) (-17,0) (-16,0) (-10,0) (0,0)
0
a: (-20,0) (-19,0) (-18,0) (-17,0) (-16,0) (-10,0) (-9,0) (0,0)
0
a: (-20,0) (-19,0) (-18,0) (-17,0) (-16,0) (-10,0) (-9,0) (-8,0) (0,0)
0
a: (-20,0) (-19,0) (-18,0) (-17,0) (-16,0) (-10,0) (-9,0) (-8,0) (-7,0) (0,0)
a: (-19,0) (-18,0) (-17,0) (-16,0) (-10,0) (-9,0) (-8,0) (-7,0) (0,0)
a: (-18,0) (-17,0) (-16,0) (-10,0) (-9,0) (-8,0) (-7,0) (0,0)
a: (-17,0) (-16,0) (-10,0) (-9,0) (-8,0) (-7,0) (0,0)
a: (-16,0) (-10,0) (-9,0) (-8,0) (-7,0) (0,0)
a: (-16,0) (-9,0) (-8,0) (-7,0) (0,0)
a: (-9,0) (-8,0) (-7,0) (0,0)
a: (-8,0) (-7,0) (0,0)
a: (-7,0) (0,0)
a: (-7,0)
a:
